### ISSUE TITLE
[Feature] download browsers in parallel 

### ISSUE THREAD

**Issue by aslushnikov** [microsoft/playwright#2216]
It'd be nice to download browsers in parallel rather than sequentially. In my measurements, this should give us a 20% speedup.

**Comment by pavelfeldman** [comment]
Are you sure we need this? With the cached browsers it never bothered me...

**Comment by aslushnikov** [comment]
Not pressing, for sure! But would be nice to have since it'll speedup CI's

**Comment by pavelfeldman** [comment]
Closing for now unless you want to do it right away!
